_data/twenty_20/faqs.yml

```diff
@@ -0,0 +1,22 @@
+- question: "How much does participation cost?"
+  answer: "There is no participation fee. Lunches and coffee breaks will be
+  covered."
+
+- question: "Is there anything else for free?"
+  answer: "We can offer a limited number of rooms during the school (four
+  nights). For more details, see the <a href=\"../register\">registration
+  page</a> (<em>the registration deadline was August 31</em>)."
```

Comment:
Same as above.

Reply:
This is actually just copied. Good catch :+1: